client/modules/datafiles/src/projects/utils.ts

```diff
@@ -47,10 +47,9 @@ export const getHazmapperUrl = (
  * out.
  */
 export const filterHazmapperMaps = (maps: THazmapperMap[]): THazmapperMap[] => {
-  // Filter out non-production maps if we are DS prod (i.e. 'designsafe-ci.org')
-  if (window.location.origin.includes('designsafe-ci.org')) {
+  // Filter out non-production maps if we are DS prod
+  if (window.location.origin.startsWith('https://www.designsafe-ci.org')) {
```

Comment:
The url `https://designsafe-ci.org/` also works, we should include that as an option as well

Reply:
@rstijerina Thanks! Fixed.